### PR TITLE
Provide README for the passwordExpiry tool

### PR DESCRIPTION
**Pull request by pbdeuchler**

### CODE REVIEW

File: examples/PasswordExpiryWatcher/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+3. Download the zip file of your choice 
+	- We always recommend downloading from the latest release if possible
+	- Most Windows users will want the `386` zip
+4. Extract the files from the zip
+	- Right click on the zip file and click `Extract All` on Windows
+	- Double click on the zip file on macOS/Linux
+5. `PasswordExpiryWatcher_myos_myarch` is the relevant binary
+	- Some may find this is an unwieldy name to type every time. Feel free to rename to your liking, in fact if this is a tool you plan on running frequently we encourage this for ease of use.
+
+> If you'd like to be able to call this binary from an arbitrary directory you can move it to `/usr/local/bin` on Linux and macOS
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md:23]: On Windows, you could add it to C:\Windows\System32\ or anywhere else listed in %PATH%. Dunno if you'd really want to suggest that, but it's an option.

File: examples/PasswordExpiryWatcher/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+3. Run go install
+	- `go install .`
+
+This will install a binary called `passwordExpiryWatcher` to your `$GOBIN`
+
+> If you'd like to be able to call this binary from an arbitrary directory make sure your `$GOBIN` is in your `$PATH` (linux) or `%PATH` (windows)
+
+### To Run
+
+To run this tool you will need to use what is called the "command line". On Windows this is a program called "Power Shell", on macOS it is an app called "Terminal".
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: this is kind of condescending. I'd just say something like `To run this tool you will need to use the command line. On Windows you can use PowerShell and on OSX you can use Terminal`. Also, PowerShell is one word!

File: examples/PasswordExpiryWatcher/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+For example:
+
+macOS/Linux:
+
+`./PasswordExpiryWatcher -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
+
+Windows:
+
+`./PasswordExpiryWatcher.exe -key=82105124f2979e28273d4e8dd32b2355c5012837 -output=password_expirations.csv`
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: I think you can drop the ".exe" suffix for your commands on Windows, so they can all have the exact same instructions if you want.

File: examples/PasswordExpiryWatcher/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+
+> If you have renamed your binary simply replace `PasswordExpiryWatcher` with the new name
+
+> If you installed the binary with the Go toolchain and your `$GOBIN` is in your `$PATH` or `%PATH`, or if you moved the binary to `/usr/local/bin` you can run the above command at any time on your command line excluding the "./"
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: I'm pretty sure it's `%PATH%`, not `%PATH` on Windows. And in PowerShell, I think you need to use something weird like `$env:PATH`? I'm not opening my VM to test that though :)

File: examples/PasswordExpiryWatcher/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+##### Windows Instructions
+1. Open the program `Power Shell`
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: s/Power Shell/PowerShell/g

File: examples/PasswordExpiryWatcher/README.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+2. Using the `cd` (stands for "Change Directory") command navigate to where you downloaded your binaries
+	- For example, if we downloaded and unzipped the binaries in our `Download` folder we just have to run: `cd Downloads\JumpCloudAPI_Examples_windows_386`. If we downloaded to our desktop the command will probably look something like: `cd Desktop\JumpCloudAPI_Examples_windows_386`
+	- If you used the Go install instructions and your `$GOBIN` is in your `%PATH` you can skip step 2 and go right to 3
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: See above for `%PATH` comment.

File: examples/PasswordExpiryWatcher/README.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+3. Grab your API key from the JumpCloud Admin console
+	- Click on your email on the top right hand corner to access the API Settings
+4. Run the command
+	- `./PasswordExpiryWatcher.exe -key=YOUR_API_KEY_GOES_HERE -output=CSV_FILE_OUTPUT_GOES_HERE`
+	- If you would like your CSV file to go somewhere else besides the current directory make sure you include the _full path_ of the file
+		- Good Example: `-output=C:\Users\MyUser\CSVFiles\password_expirations.csv` 
+		- Bad Example `-output=..\CSVFiles\password_expirations.csv`
```
Comment (beckjake) [inline examples/PasswordExpiryWatcher/README.md]: The fact that this is an issue probably indicates that we're doing something wrong in our code. When tools have this issue it drives me nuts (looking at you, OSX).

File: examples/PasswordExpiryWatcher/README.md (hunk 1 above)
Comment (pbdeuchler) [inline examples/PasswordExpiryWatcher/README.md:23]: My thinking is that people who can connect those dots are probably the only people who will want to actually have the tool in their `PATH`, and if not I'd rather have support walk them through it than have them mess something up thanks to my README

**Comment by beckjake** [comment]
:+1:

**Comment by kusold** [comment]
:+1:

**Pull request merged**